test/unit/alignment/matrix/edit_distance_matrix/edit_distance_trace_matrix.hpp

```diff
@@ -0,0 +1,42 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2020, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2020, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#include <seqan3/alignment/matrix/edit_distance_trace_matrix_full.hpp>
+#include <seqan3/alignment/matrix/row_wise_matrix.hpp>
+
+#include "../../pairwise/fixture/alignment_fixture.hpp"
+
+using namespace seqan3;
+using namespace seqan3::test::alignment::fixture;
+
+using word_type = uint8_t;
+
+template <bool is_semi_global, bool use_max_errors>
+class matrix_type
+    : public seqan3::detail::edit_distance_trace_matrix_full<word_type, is_semi_global, use_max_errors>
```

Comment:
💅 please put the colon on the previous line.

Reply:
I will fix that in the followup PR.